core/state_processor.go

```diff
@@ -1111,8 +1112,8 @@ func Re(receipts types.Receipts) ([]byte, error) {
 
 func TransferTxAsMessage(tx *transactions.Transfer, baseFee *big.Int) (types.Message, error) {
 	bigIntGasPrice := big.NewInt(int64(tx.GasPrice()))
```

Comment:
bigIntGasPrice no need to be derived from GasPrice if we are passing baseFee. Assume that baseFee will not be nil as we will be using DynamicFeeTxn